src/views/Practice/Practice.jsx

```
@@ -22,84 +22,84 @@ import { MessageDialog } from "../../components/Assesment/Assesment";
+import Or from "../../assets/audio/Or.wav";
+import Ange from "../../assets/audio/ange.wav";
+import Ta from "../../assets/audio/Ta.wav";
+import Ble from "../../assets/audio/ble.wav";
+import Le from "../../assets/audio/Le.wav";
+import Mon from "../../assets/audio/mon.wav";
+import Bas from "../../assets/audio/Bas.wav";
+import Ket from "../../assets/audio/ket.wav";
+import Tun from "../../assets/audio/Tun.wav";
+import Nel from "../../assets/audio/nel.wav";
+import Sun from "../../assets/audio/Sun.wav";
+import Set from "../../assets/audio/set.wav";
```

Biome (1.9.4)

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)